frontend/src/modules/integration/integration-store.js

```diff
@@ -269,20 +269,14 @@ export default {
           setupAction,
         );
 
-        commit('CREATE_SUCCESS', integration);
-        Message.success(
-          'The first activities will show up in a couple of seconds. <br /> '
-          + '<br /> This process might take a few minutes to finish, depending on the amount of data.',
-          {
-            title: 'GitHub integration created successfully',
-          },
-        );
```

Comment:
Won't you need the success message? Asking because it's in the designs https://www.figma.com/file/9WvDkCdPAE3Yv86jj410ob/EXP%2FDrafts?type=design&node-id=590%3A5932&mode=design&t=1x5XCOBqDzMy6eHQ-1